testproject/Packages/packages-lock.json

```diff
@@ -116,42 +116,39 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.services.authentication": {
-      "version": "0.3.1-preview",
+      "version": "0.5.0-preview",
       "depth": 0,
       "source": "registry",
       "dependencies": {
-        "nuget.moq": "1.0.0",
         "com.unity.nuget.newtonsoft-json": "2.0.0",
-        "com.unity.services.core": "0.2.0-preview",
+        "com.unity.services.core": "1.1.0-pre.2",
         "com.unity.modules.unitywebrequest": "1.0.0"
       },
       "url": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates"
     },
     "com.unity.services.core": {
-      "version": "0.2.0-preview",
-      "depth": 1,
+      "version": "1.1.0-pre.2",
+      "depth": 0,
       "source": "registry",
       "dependencies": {
-        "nuget.moq": "1.0.0",
         "com.unity.modules.unitywebrequest": "1.0.0",
         "com.unity.nuget.newtonsoft-json": "2.0.0"
       },
       "url": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates"
     },
     "com.unity.services.relay": {
-      "version": "0.0.1-preview.3",
+      "version": "file:../../../com.unity.services.relay/com.unity.services.relay",
```

Comment:
I think this file, `packages-lock.json`, also needs to be reverted. It contains path to local repos, which makes me think it was also commited by mistake.